source/the-exegol-project/python-wrapper.rst

```diff
@@ -38,6 +38,7 @@ Below is a, non-exhaustive, list of what the wrapper supports:
 :ref:`Env. variables<feature_env>`                  Support for environment variable configuration
 :ref:`Device sharing<feature_device_sharing>`       Support for hardware sharing
 :ref:`Custom hostname<feature_hostname>`            Support for customizing a specific container hostname
+:ref:`Comments <feature_comment>`                    User can add any text comments to the container
```

Comment:
I think the trailing space after between "Comments" and "<" needs to be removed to avoid any risk of bad RST parsing

Reply:
There is no trailing space, it's just the github / mobile preview that make it weird